lang/libcxx/Portfile

```diff
@@ -88,64 +87,24 @@ if {${os.major} < 11 || [variant_isset replacemnt_libcxx]} {
 
     supported_archs i386 x86_64
 
-    if {${build_arch} in "i386 x86_64"} {
+    # 10.4 supports i386 only
```

Comment:
I didn't have any particular problem building libcxx as universal i386/x86_64 on Tiger Intel.

[libcxx-5.0.1_4+universal.darwin_8.i386-x86_64.tbz2](http://cunningham.homeunix.com:6227/libcxx/libcxx-5.0.1_4+universal.darwin_8.i386-x86_64.tbz2)

Reply:
so I guess we could remove that part about 10.4 supporting i386 only.

Reply:
of course, clang11-bootstrap is not going to build on Tiger, so this PR will pretty much nuke that anyway. Does clang11-bootstrap build on Leopard?

Reply:
I haven't tested it yet. Anyway, idea to revert this line seems good.